test/_routes.js

```diff
@@ -125,5 +125,103 @@ module.exports = function () {
           done()
         })
     })
+
+    test('serve routes with a * wildcard', function (done) {
+      const app = express()
+
+      const oapi = openapi()
+      app.use(oapi)
+      app.get(['/route/:param*'], oapi.path({
```

Comment:
Is the array usage in this test here intentional? Might be nice to remove it you dont specifically intent to test array handling behavior for clarity.

Reply:
I will remove the array, thanks for catching this!